app/medlem/[privatePage]/layout.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const MedlemPageLayout = ({ children }: { children: React.ReactNode }) => (
+  <div className="min-h-screen">
```

Comment:
This layout is duplicated for medlem page, content page & post(s), could be extracted to some PageLayout component